src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -271,6 +272,12 @@
         + "renaming map produced should be saved")
     private String variableMapOutputFile = "";
 
+    @Option(name = "--instrument_param_mapping_report",
+        usage = "File where the encoded parameters created by Production "
+            + "Instrumentation are mapped to their pre-encoded values. Must "
+            + "be used in tandem with --instrument_code=PRODUCTION")
+    private String paramMappingOutputFile = "";
```

Comment:
Let's name this variable similar to flag name. It should at least mention `instrumentation` in it otherwise it's too generic.

Reply:
Changed to `instrumentationMappingOutputFile`